images/windows/toolsets/toolset-2022.json

```diff
@@ -264,7 +264,17 @@
             "Component.MDD.Linux",
             "Component.MDD.Linux.GCC.arm",
             "Component.Microsoft.Windows.DriverKit",
-            "wasm.tools"
+            "wasm.tools",
```

Comment:
Do we really need these components?

Reply:
Yes! . These components are mandatory for latest WDK.